### DISCUSSION TITLE
我好像发现了法厄同的据点

### DISCUSSION THREAD

**Discussion by reversed-PAI** [share121/inter-knot#229] [其他]
![IMG_20240809_221606](https://github.com/user-attachments/assets/d34c2920-ae91-4722-82c0-ebf31bb64edd)
六分街那家叫Random Play的录像店，里面有扇门不知道大家发现没，门窗上趴着一只邦布似乎只有店长来的时候才会开门，有次店长进去的时候忘关门了，我瞄了一眼，里面有一面电视墙！似乎就是传说中的HDD。听说只有法厄同才有这种装备，这里会不会就是传奇绳匠法厄同的据点？

**Comment by SDCOM-0415** [comment]
我进去过，铃很热情的招待了我，里面只是普通的一些录像带播放用的东西而已，里面也没有其他的了

**Comment by 102266270** [comment]
我是六分街的居民可以作证，这只是一个普通的录像店，绝对不是法厄同的据点！！！

**Comment by NtskeyTeams** [comment]
这个委托我接了！
